### PR TITLE
dont allow deleting a security question if users have already used it

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by saeedmb8** [comment]
screen shot 2017-11-06 at 12 56 33

**Comment by smarzban** [comment]
screen shot 2017-11-06 at 13 07 23

File: molo/profiles/views.py [hunk 1]
```diff
@@ -223,14 +223,19 @@ def form_valid(self, form):
...
+            try:
                 saved_answer = user.profile.securityanswer_set.get(
                     user=user.profile,
                     question=self.security_questions[i]
                 )
-                answer_checks.append(
-                    saved_answer.check_answer(user_answer)
-                )
+                answer_checks.append(saved_answer.check_answer(user_answer))
+            except SecurityAnswer.DoesNotExist:
```
Comment (codiebeulaine) [inline molo/profiles/views.py:233]: We should perhaps be more specific and say you have not answered any security questions?

Comment (codiebeulaine) [reply molo/profiles/views.py:233]: Also, if we do this, we should give them the option to answer the questions if they have not done so already

**Comment by codiebeulaine** [comment]
@saeedmb8 👍 although we need to give them the opportunity to set or reset their security questions after registrations asap

**Comment by smarzban** [comment]
@codiebeulaine agree and yes, we are going to add the option to change or update their security question in their profile

**Pull request merged**